### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sebastien-marichal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
 
 ### Overview
```
Comment (pavel-mikula-sonarsource) [inline README.md:15]: 2022 in license header above should get updated

File: README.md [hunk 2]
```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
...
 This repo contains tools to help integrate Roslyn analyzers with SonarQube so that issues detected by the Roslyn analyzers are reported and managed in SonarQube.
-Specifically, the tools will generate a Java SonarQube plugin that registers the rules with SonarQube. The generated plugin works with the [C# plugin](http://docs.sonarqube.org/x/bAAW) (v4.5 or higher) and the [SonarQube Scanner for MSBuild](http://docs.sonarqube.org/x/Lx9q) (v2.0 or higher) to handle executing the analyzer and uploading any issues.
+Specifically, the tools will generate a Java SonarQube plugin that registers the rules with SonarQube. The generated plugin works with the [C# plugin](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/languages/csharp/) (v4.5 or higher) and the [SonarQube Scanner for MSBuild](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/) (v2.0 or higher) to handle executing the analyzer and uploading any issues.
```
Comment (pavel-mikula-sonarsource) [inline README.md:17]: What does the `v4.5` refers to?

Comment (pavel-mikula-sonarsource) [inline README.md:17]: I would probably remove the `(v2.0 or higher)` commitment for the backward compatibility. 
I don't think users really care, and we want everyone to be on the latest anyway

File: README.md [hunk 3]
```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
...
 See this [blog post](https://devblogs.microsoft.com/devops/sonarqube-scanner-for-msbuild-v2-0-released-support-for-third-party-roslyn-analyzers/) for more information.
 
 #### Integration with the SonarQube Scanner for MSBuild
-The [SonarQube Scanner for MSBuild](http://docs.sonarqube.org/x/Lx9q) will automatically execute your custom rules as part of an analysis build using the configuration you have defined in the Quality Profile. There is no need to manually reference your analyzer NuGet package in the MSBuild projects you want to analyse.
+The [SonarQube Scanner for MSBuild](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner-for-dotnet/) will automatically execute your custom rules as part of an analysis build using the configuration you have defined in the Quality Profile. There is no need to manually reference your analyzer NuGet package in the MSBuild projects you want to analyse.
```
Comment (pavel-mikula-sonarsource) [inline README.md:21]: It's not called "SonarQube Scanner for MSBuild", it's called "Scanner for .NET"

File: README.md [hunk 4]
```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
...
 
 #### Integration with SonarLint for Visual Studio
 If you define a Quality Profile that references your custom rules then [SonarLint for Visual Studio ](https://github.com/sonarsource/sonarlint-visualstudio) in *Connected Mode* will include those rules in the ruleset it generates.
 However, it will **not** automatically configure Visual Studio to execute your custom rules.
 If you want your customer rules to be executed inside Visual Studio then you will need reference your analyzer NuGet package from your MSBuild projects, or install your analyzer VSIX on each developer machine.
 
-See the [SonarLint for Visual Studio wiki](https://github.com/SonarSource/sonarlint-visualstudio/wiki/Connected-Mode) for more information on Connected Mode.
+See the [SonarLint for Visual Studio wiki](https://docs.sonarsource.com/sonarlint/visual-studio/team-features/connected-mode/) for more information on Connected Mode.
```
Comment (pavel-mikula-sonarsource) [inline README.md:30]: It's not wiki anymore

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Some polishing

File: README.md [hunk 5]
```diff
@@ -14,23 +14,23 @@ Licensed under the [GNU Lesser General Public License, Version 3.0](http://www.g
...
 
-The SonarQube Scanner for MSBuild can also import issues from Roslyn analyzers that do not have plugins created using this SDK. However, they will be imported as "external issues" and are handled differently in SonarQube. See [Importing Issues from Third-Party Roslyn Analyzers (C#, VB.NET)](https://docs.sonarqube.org/pages/viewpage.action?pageId=11640944) for more information.
+The SonarQube Scanner for MSBuild can also import issues from Roslyn analyzers that do not have plugins created using this SDK. However, they will be imported as "external issues" and are handled differently in SonarQube. See [External .NET issues](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/importing-external-issues/external-analyzer-reports/#external-dotnet-issues) for more information.
 
 #### Integration with SonarLint for Visual Studio
 If you define a Quality Profile that references your custom rules then [SonarLint for Visual Studio ](https://github.com/sonarsource/sonarlint-visualstudio) in *Connected Mode* will include those rules in the ruleset it generates.
```
Comment (pavel-mikula-sonarsource) [inline README.md:26]: There was a PR for this
```suggestion
If you define a Quality Profile that references your custom rules then [SonarLint for Visual Studio](https://github.com/sonarsource/sonarlint-visualstudio) in *Connected Mode* will include those rules in the ruleset it generates.
```

File: README.md (hunk 2 above)
Comment (sebastien-marichal) [reply README.md:17]: I am assuming the C# Analyzer version.
It is not clear what is used to link to.
Assuming it is about our C# plugin, I think I can drop the compatibility version as for S4NET.

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**